plugins/inputs/reader/dev/telegraf.conf

```diff
@@ -0,0 +1,106 @@
+
```

Comment:
Slim this file down by removing comments and default values. See [other example](https://github.com/influxdata/telegraf/pull/4324/files#diff-6076d926ad050faf6d393602b4581adb).